src/metabase/pulse/render.clj

```diff
@@ -146,11 +179,12 @@
       (if (:card-error (ex-data e))
         (do
           (log/error e (trs "Pulse card query error"))
-          (body/render :card-error nil nil nil nil nil))
+          (assoc (body/render :card-error nil nil nil nil nil)
+                 :attachments (error-file-attachment card e)))
         (do
           (log/error e (trs "Pulse card render error"))
-          (body/render :render-error nil nil nil nil nil))))))
-
+          (assoc (body/render :render-error nil nil nil nil nil)
+                 :attachments (error-file-attachment card e)))))))
```

Comment:
I create the attachment here because I want access to the Exception and its stacktrace.